storage_service/locations/tests/test_fixity_log.py

```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+
+from locations import models
+
+
+class TestFixityLog(TestCase):
+
+    fixtures = ['base.json', 'fixity_log.json']
```

Comment:
This fixture fails to load with `IntegrityError: Problem installing fixtures: The row in table 'locations_fixitylog' with primary key '1' has an invalid foreign key: locations_fixitylog.package_id contains a value 'e0a41934-c1d7-45ba-9a95-a7531c063ed1' that does not have a corresponding value in locations_package.uuid.`  You need to add the 'package.json' to the fixtures list.